content/Encapsulation/Encapsulation.facts.md

```diff
@@ -0,0 +1,67 @@
+### Facts for Encapsulation lesson:
+
+`Encapsulation` is a way of wrapping the properties and methods of a class together as a single unit. It hides the internal implementation of an object from other objects for protection purposes.
+
+The main purpose of encapsulation is to reduce software development complexity and to protect the internal state of an object.
+ 
+`Encapsulation` is achieved by keeping each object state `private` inside a class that restricts other objects to directly access this state.
+
+An object is not allowed to directly access and change its properties. It can only access and change the values of its properties through `public` methods.
```

Comment:
@clydiex Thank you for the suggestion. The reason why I didn't mention any `accessor` or `mutator` term here in the course is to avoid providing information that does not exists or used in the PHP manual. I have read some resources about getters and setters but it is not emphasized in the manual so I opted not to introduce/include that info in the lessons. Hope this helps. Thanks.

Reply:
Ok, noted. Thank you.